src/services/main.service.ts

```diff
@@ -128,18 +128,23 @@ export class MainService {
   }
 
   private async loadBlockModules(block: BlockMapping) {
-    const status = block.element.dataset.blockStatus;
+    const status = block.element.dataset.blockStatus ?? '';
 
-    if (status !== 'loading' && status !== 'loaded') {
+    if (!['loading', 'loaded', 'error'].includes(status)) {
```

Comment:
ich würde das in eine funktion speichern und die states vielleicht in eine property auslagern

Reply:
Ich hab die logic etwas angepasst. Eventuell passt es jetzt. 
@samuelecalamita